### PR TITLE
PRIME-1042 Complete Approve Decline Site Registration Buttons

### PR DESCRIPTION
**Pull request by mtpultz**

### CODE REVIEW

File: prime-dotnet-webapi/Migrations/20200911200512_AddedSiteStatus.cs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "Status",
+                table: "Site",
+                nullable: false,
+                defaultValue: 0);
+
+            migrationBuilder.CreateTable(
+                name: "NewestAgreements",
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Migrations/20200911200512_AddedSiteStatus.cs:18]: Oh dear

File: prime-dotnet-webapi/Migrations/20200911200512_AddedSiteStatus.cs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                    Id = table.Column<int>(nullable: false)
+                        .Annotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.IdentityByDefaultColumn)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_NewestAgreements", x => x.Id);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "SiteRegistrationNote",
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Migrations/20200911200512_AddedSiteStatus.cs:30]: I looks like this migration was made off a stale ApiDbContextModelSnapshot, it doesn't include changes that Nathan made

File: prime-dotnet-webapi/Migrations/20200911200512_AddedSiteStatus.cs (hunk 1 above)
Comment (mtpultz) [reply prime-dotnet-webapi/Migrations/20200911200512_AddedSiteStatus.cs]: haha nooooo

**Review by james-hollinger — APPROVED** [review]
LGTM

**Review by AnaisHebert — APPROVED** [review]
🥞🍳

**Pull request merged**